cmake/CallbackSetupExtraOptions.cmake

```diff
@@ -3,29 +3,6 @@
 # TrilinosCreateClientTemplateHeaders.cmake
 SET(CMAKE_MODULE_PATH  ${CMAKE_MODULE_PATH} "${Trilinos_SOURCE_DIR}/cmake")
 
-MACRO(TRILINOS_DISABLE_PACKAGE_REQUIRING_CXX11  CXX11_PACKAGE_NAME_IN)
```

Comment:
Right, since C++11 has been required for a long time, this macro should have been deleted a long time ago.